packages/runtime/package.json

```diff
@@ -52,7 +52,7 @@
     "pre-deploy": "./scripts/pre-deploy.sh",
     "generate-release-notes": "ts-node ./scripts/generate-release-notes.ts",
     "deploy:update-disclaimer": "yarn licenses generate-disclaimer --prod > src/account/LicenseDisclaimer.txt && ./scripts/copy_license_to_android_assets.sh",
-    "postinstall": "patch-package && yarn keys:decrypt && ./scripts/copy_license_to_android_assets.sh",
+    "//postinstall": "patch-package && yarn keys:decrypt && ./scripts/copy_license_to_android_assets.sh",
```

Comment:
Disabled it since we don't want that here.

Note: we'll remove most of these scripts after the modular runtime becomes the "source" of truth for apps. But keeping for now to avoid future conflicts.